grails-app/domain/com/unifina/domain/community/CommunitySecret.groovy

```diff
@@ -23,15 +23,17 @@ class CommunitySecret {
 	}
 
 	@GrailsCompileStatic
-	Map toMap() {
-		// TODO:
-		//  We probably need to be able to return the secret in the object?
-		//  Especially if it's generated server-side.
-		//  The API should of course only serve the secrets to the community admin.
-		return [
-			id: id,
-			name: name,
-			communityAddress: communityAddress,
-		]
-	}
+	Map toMap() { [
```

Comment:
`toMap(boolean includeSecret = false)` would be more DRY and still be backwards compatible?